### DISCUSSION TITLE
Problems compiling ImageMagic on Fedora 33 / libx265

### DISCUSSION THREAD

**Discussion by polemon** [ImageMagick/ImageMagick#2976] [Help]
Hi, I just pulled the latest trunk from github, but when compiling I'm getting a weird error, and I'm not quite sure it's on my end:
While compiling I'm not really getting any hard errors, but once compiled and installed, 
`magick: error while loading shared libraries: libx265.so.169: cannot open shared object file: No such file or directory`

When inspecting the binary with `ldd` I see this:
`libx265.so.169 => not found`

I'm not sure why it's linking to `libx265.so.169`, a later version is installed:
```
$ locate libx265
/usr/lib64/libx265.so
/usr/lib64/libx265.so.192
/usr/lib64/libx265_main10.so.192
/usr/lib64/libx265_main12.so.192
```

The library is installed from the regular Fedora repos. I'm not sure if this is a problem on my end, but in any event, is there a way I can force linking to `libx265.so.192`?

**Comment by urban-warrior** [comment]
You may need to refresh the LD cache (e.g. ldconfig).  We build ImageMagick under Fedora 33 daily and have not encountered this problem.

**Reply by polemon** [reply]
OK, I deleted `/etc/ld.so.cache` and the re-generated it with `ldconfig` but I'm still getting the exact same error.
`ldconfig -p` shows:
```
	libx265.so.192 (libc6,x86-64) => /lib64/libx265.so.192
	libx265.so (libc6,x86-64) => /lib64/libx265.so
```
`/lib65/libx265.so` is a link to the correct file:
```
ll /lib64/libx265.so           
lrwxrwxrwx 1 root root 14 Aug 19 03:29 /lib64/libx265.so -> libx265.so.192
```
Hmm, I can't figure out what I'm doing wrong, perhaps I should take this to StackExchane.
